src/components/SummaryContentForm.tsx

```
@@ -0,0 +1,44 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { useTheme } from 'next-themes';
+import React, { useState } from 'react';
+import TextEditor from '~src/ui-components/TextEditor';
+
+interface Props {
+	className?: string;
+	height?: number;
+	onChange?: (content: string) => void | string | null;
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)